dbflute-engine/embedded/templates/doc/html/diffmodel.vm

```diff
@@ -16,961 +16,968 @@
 <!DOCTYPE html>
 <html lang="en"> <!-- main language (to avoid HTML validator) -->
 <head>
-	<meta charset="UTF-8" />
-	<title>${database.projectName} ${selector.diffModelSmallTitle}</title>
```

Comment:
Big difference exists in the file.
Is it by auto-formatter?
Please don't change the format of unrelated lines.

Reply:
sorry, I fixed it.